.github/workflows/check_pr_cklib.yml

```diff
@@ -38,3 +38,9 @@ jobs:
       - name: Run tests
         working-directory: ./cklib
         run: tox
+
+      - name: Archive code coverage results
+        uses: actions/upload-artifact@v2
+        with:
+          name: cklib-code-coverage-report
+          path: cklib/htmlcov/
```

Comment:
what do we do with those reports?

Reply:
I used them to figure out the missing 2% to get 100% coverage. 😄